core/controllers/admin.py

```diff
@@ -1160,6 +1161,73 @@ def get(self):
         })
 
 
+class TranslationContributionStatsHandler(base.BaseHandler):
+    """Handler to show the translation contribution stats of a user."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_access_admin_page
+    def get(self):
+        username = self.request.get('username', None)
+        if username is None:
+            raise self.InvalidInputException('Missing username param')
```

Comment:
Please add schema for the handler and remove this check* (I'm assuming we are not allowing new handler without schema)